content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/expiration_dates.rst

```diff
@@ -1,65 +1,230 @@
+===================================
+
+.. note::
+  To sell perishable products with expiration dates, the :guilabel:`Removal Strategy` for the
+  :guilabel:`Location` the products are stored in must be set to
+  :abbr:`FEFO (First Expiry, First Out)`. If there is not enough stock of perishable products in
+  one lot, Odoo will automatically take the remaining quantity required from a second lot with the
+  next-soonest expiration date. Removal strategies can also be set on
+  :guilabel:`Product Categories`.
+
+.. seealso::
+    - :doc:`../../routes/strategies/removal`
+
+Selling perishable products with expiration dates is done the same as any other type of product.
+The first step in selling perishable products is to create a sales order. To do that, go to
+:menuselection:`Sales --> Create` to create a new quotation, and fill out the information. Add a
+:guilabel:`Customer`, click :guilabel:`Add a product` to add products to the :guilabel:`Product`
+lines, then set a :guilabel:`Quantity`.
+
+Then, click the :guilabel:`Other Info` tab. Under the :guilabel:`Delivery` section, change the
+:guilabel:`Delivery Date` to a date after the expected date, and click the
+:guilabel:`green checkmark` to confirm the date. Finally, :guilabel:`Confirm` the sales order.
+
+Click the :guilabel:`Delivery` smart button at the top of the sales order to be taken to the
+:guilabel:`Warehouse Receipt Form`. Click :guilabel:`Validate` and :guilabel:`Apply` to process all
+:guilabel:`Done` quantities and deliver the products. If the products are delivered before
+the :guilabel:`Alert Date` set on the product form, then no alerts will be created.
+
+View expiration dates for lots & serial numbers
+===============================================
+
+To view and group all products with expiration dates by lot number, go to :menuselection:`Inventory
+--> Products --> Lots/Serial Numbers`. Remove any :guilabel:`Filters` in the
+:guilabel:`Search bar`, then click :guilabel:`Group By`, :guilabel:`Add Custom Group`, and click
+the :guilabel:`drop-down` to select :guilabel:`Expiration Date.` :guilabel:`Apply` the filter. This
+will break down all perishable products, their :guilabel:`Expiration Dates`, and the assigned lot
+number.
+
+.. image:: expiration_dates/expiration-dates-group-by-dates.png
    :align: center
+   :alt: Group by expiration dates on lots and serial numbers page.
+
+To see any expiration alerts for products that are either expired (or will expire soon), remove all
+:guilabel:`Filters` from the :guilabel:`Search bar`. Then, click :guilabel:`Filters`, then
+:guilabel:`Expiration Alerts`.
+
+Expiration alerts
+-----------------
 
-Expiration Alerts
-=================
+To see expiration alerts, go to :menuselection:`Inventory --> Products --> Lots/Serial Numbers`.
+Then, click into a :guilabel:`Lot/Serial Number` containing perishable products. Once on the serial
+number form, click the :guilabel:`Dates` tab to see all expiration information about the products.
```

Comment:
To see expiration alerts, go to :menuselection:`Inventory application --> Products --> Lots/Serial Numbers`.

Then, click into a :guilabel:`Lot/Serial Number` with perishable products. Doing so reveals the serial
number detail form. On the serial number detail form, click the :guilabel:`Dates` tab to see all expiration information related to the products.